docs/cheat-sheet.md

```diff
@@ -28,16 +28,16 @@ The cheat sheet provides simple code examples of many of MLRun's features.
 - [Hyperparameter tuning](#hyperparameter-tuning)
 
 ## MLRun setup
-Docs: [Set up your client environment](./install/remote.md), [Installation and setup guide](./install.md)
+Docs: [Set up your client environment](./setup-guide/remote.md), [Development Guide](setup-guide.md)
 
-### MLRun server/client overview
+### MLRun server/client overviewß
 
 MLRun has two main components, the service and the client (SDK+UI):
-- MLRun service runs over Kubernetes (can also be deployed using local Docker for demo and test purposes) - see [installation documentation](./install.md) for more information
+- MLRun service runs over Kubernetes (can also be deployed using local Docker for demo and test purposes) - see [MLRun CE installation documentation](setup-guide.md) for more information
```

Comment:
- MLRun service runs over Kubernetes (can also be deployed using local Docker for demo and test purposes). See the [MLRun CE installation documentation](./setup-guide.md) for more information.